Frontend/src/Components/ListingItem/ListingItem.jsx

```diff
@@ -6,8 +6,21 @@ const API_KEY = import.meta.env.VITE_BACKEND_ADDRESS;
 
 const ListingItem = ({ id, title, price, imageUrls }) => {
```

Comment:
since we pass in {...listings} as a prop here https://github.com/tobygabriella/worldcollective/blob/cdc54e0af71eec71f5676312709ac0a5732ba1f1/Frontend/src/Components/SearchResults/SearchResults.jsx#L77 for example, is it possible to pass in status and liked from everywhere you call ListingItem? just to avoid fetching from the API every time